source/features/make-discussion-sidebar-sticky.js

```diff
@@ -0,0 +1,14 @@
+import select from 'select-dom';
+import debounce from 'debounce-fn';
+
+function updateStickiness() {
+	const sidebar = select('.discussion-sidebar');
+	const sidebarHeight = sidebar.offsetHeight + 25;
+	const windowHeight = window.innerHeight;
+	sidebar.style.position = sidebarHeight < windowHeight ? 'sticky' : null;
+}
+
+export default function () {
+	updateStickiness();
+	window.onresize = debounce(updateStickiness, {wait: 100});
```

Comment:
`addEventListener`

Also this listener will have to be removed on the next ajaxLoad, which complicates things. Look at how `copy-on-y` is implemented (in short: `enableFeature` goes to `onDomReady` and this file has to handle its own [`onAjaxedPages` + `pageDetect`](https://github.com/sindresorhus/refined-github/blob/40ff3079bea59a83df1d56220e4f1e9f5e0f43de/source/features/copy-on-y.js#L13-L15))

Reply:
Okay. I used `.onresize` because `delegate` didn't work, but that's obviously the better solution.